### PR TITLE
Added a documentation to recalculate cinis and its wizard.

### PR DESCRIPTION
**Pull request by polsala**
Updated distri/adm-pub/cnmc.md docs:

- Added documentation.
- Added translations.

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_new_cini_wizard_docu/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_new_cini_wizard_docu/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_new_cini_wizard_docu/

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_new_cini_wizard_docu/

File: locales/es_ES/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -16616,6 +16616,122 @@ msgstr "CINI"
...
+"se puede hacer desde el asistente correspondiente, este permite actualizar\n"
+"tanto los códigos TI como los CINI. "
+
+msgid "Localització de l'assistent"
+msgstr "Localización del asistente"
+
+msgid ""
+"<strong> Administració pública > CNMC > Circulars > Recalcular CINIs / Tis </strong>"
+msgstr ""
+"<strong> Administración pública > CNMC > Circulars > Recalcular CINIs / Tis </strong>"
```
Comment (jaumef) [inline locales/es_ES/LC_MESSAGES/messages.po]: Fix translations
Circulars -> Circulares

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_new_cini_wizard_docu/

**Pull request merged**